airbyte-integrations/connectors/source-zendesk-chat/source_zendesk_chat/manifest.yaml

```diff
@@ -58,10 +58,10 @@ definitions:
       type: RequestOption
       field_name: limit
     pagination_strategy:
-      type: CustomPaginationStrategy
-      class_name: source_zendesk_chat.components.time_offset_pagination.ZendeskChatTimeOffsetIncrementPaginationStrategy
-      time_field_name: end_time
+      type: CursorPagination
       page_size: 1000
+      cursor_value: '{{ response.get("end_time", {}) }}'
+      stop_condition: "{{ last_page_size < 1_000 }}"
```

Comment:
maybe: last_page_size < page_size?

Reply:
yeah, but we do not pass page_size to the jinja interpolation, that's why i had to pass it as int 1_000